pybliometrics/scopus/utils/startup.py

```diff
@@ -60,16 +66,23 @@ def get_config() -> Type[ConfigParser]:
                                 'https://pybliometrics.readthedocs.io/en/stable/configuration.html')
     return CONFIG
 
-
-def get_config_path() -> Type[Path]:
-    """Function to get the configuration file path."""
-    return CONFIG_DIR or CONFIG_FILE
-
-
 def get_keys() -> List[str]:
     """Function to get the API keys and overwrite keys in config if needed."""
     if CUSTOM_KEYS:
         keys = CUSTOM_KEYS
     else:
         keys = [k.strip() for k in CONFIG.get('Authentication', 'APIKey').split(",")]
     return keys
+
+def get_folder(api, view):
+    """Auxiliary function to get the cache folder belonging to an API,
+    eventually create the folder.
+    """
+    parent = Path(CONFIG.get('Directories', api))
+    # Create folder if it does not exist
+    try:
+        folder = parent/view
+    except TypeError:
+        folder = parent
+    folder.mkdir(parents=True, exist_ok=True)
```

Comment:
The idea is to move the creation of folders into `int()`. That is, read out the paths, iterate through all of them and create them if necessary, and add them to the configuration file if necessary. And at the place where `get_folder()` is currently used, just use `parent = Path(CONFIG.get('Directories', api))` and then `folder = parent/view`. At this point we can rely on the fact that `init()` made sure all the relevant information are there.

Reply:
Understood. Should I also create all the 'view' folders with the `init()`?

Reply:
No that won't work bc that this state the allowed views are not yet known.
So we actually still need the `.mkdir()` in the `Base()` class. Good that asked